client/src/components/web-extension/Common/AuthLogin/index.jsx

```diff
@@ -63,56 +64,61 @@ export const AuthLogin = ({
-					password: '',
-				}}
-				validate={onValidate}
-				onSubmit={handleFormSubmit}
-			>
-				{({ errors, touched, setFieldValue, handleBlur, handleSubmit }) => {
-					return (
-						<div className="cd_we_create-wallet-layout--body">
-							<Form noValidate onSubmit={handleSubmit}>
-								{header}
-								<Form.Group className="mb-3">
-									<Form.Label>{passwordLabel}</Form.Label>
-									<FormControl
-										onBlur={handleBlur}
-										onChange={(e) => onChangeHandler(e, setFieldValue)}
-										name="password"
-										type="password"
-										placeholder="Enter password"
-									/>
-									{errors.password && touched.password && (
-										<Form.Text className="invalid-feedback" id="passwordHelpBlock">
-											{errors.password}
-										</Form.Text>
-									)}
-								</Form.Group>
-								{serverErrors && (
-									<Form.Text className="invalid-feedback" id="passwordHelpBlock">
-										{serverErrors.message}
-									</Form.Text>
-								)}
-								<div className="cd_we_page--bottom">
-									<Button type="submit" className="cd_we_btn-next" disabled={false}>
-										Unlock
+		<Formik
+			initialValues={{
+				password: '',
+			}}
+			validate={onValidate}
+			onSubmit={handleFormSubmit}
+		>
+			{({ errors, touched, setFieldValue, handleBlur, handleSubmit }) => {
+				return (
+					<Form noValidate onSubmit={handleSubmit}>
+						{header}
+						<Form.Group className="mb-3">
+							<Form.Label>{passwordLabel}</Form.Label>
+							<FormControl
+								onBlur={handleBlur}
+								onChange={(e) => onChangeHandler(e, setFieldValue)}
+								name="password"
+								type="password"
+								placeholder="Enter password"
+							/>
+							{errors.password && touched.password && (
+								<Form.Text className="invalid-feedback" id="passwordHelpBlock">
+									{errors.password}
+								</Form.Text>
+							)}
+						</Form.Group>
+						{serverErrors && (
+							<Form.Text className="invalid-feedback" id="passwordHelpBlock">
+								{serverErrors.message}
+							</Form.Text>
+						)}  
+						<div className="cd_we_page--bottom">
+							<Button type="submit" className="cd_we_btn-next" disabled={false}>
+								Unlock
+							</Button>
+							{isShowReset && (
+								<div className="cd_we_welcomeBack--bottom">
+									<Button onClick={handleOnReset} variant="link">
+										Log in as another user?
 									</Button>
-									{isShowReset && (
-										<div className="cd_we_welcomeBack--bottom">
-											<Button onClick={handleOnReset} variant="link">
-												Log in as another user?
-											</Button>
-										</div>
-									)}
 								</div>
-							</Form>
+							)}
 						</div>
-					);
-				}}
-			</Formik>
-		</div>
+					</Form>
+				);
+			}}
+		</Formik>
 	);
 };
+
+AuthLogin.propTypes = {
+	header: PropTypes.node,
+	isShowReset: PropTypes.bool,
+	onLoginSuccess: PropTypes.func,
+	passwordLabel: PropTypes.string
+}
+
+export default AuthLogin;
```

Comment:
We should always have a default export, usually it's used for the file component and easier to import from external component.
Named export is good for smaller utils.